X16 Reference - 11 -  Working with CMDR-DOS.md

```diff
@@ -0,0 +1,368 @@
+# Chapter 11: Working With CMDR-DOS
+This manual describes Commodore DOS on FAT32, aka CMDR-DOS. 
+
+## CMDR-DOS
+
+Commander-16 duplicates and extends the programming interface used by Commodore's line of disk drives, including the 
```

Comment:
I think we should be consistent on the naming of the product.

Reply:
Fixed.